ansible/roles/synapse/tasks/main.yml

```diff
@@ -29,8 +29,8 @@
           cert-manager.io/cluster-issuer: "letsencrypt-prod"
           external-dns.alpha.kubernetes.io/hostname: "{{ matrix.server_name }}"
         tls:
-          - hosts :
-            - "{{ matrix.server_name }}"
+          - hosts:
+              - "{{ matrix.server_name }}"
```

Comment:
Not sure that the indentation introduced by the linter will not mess things up  with this value